drivers/sensor/icm42605/icm42605_setup.c

```diff
@@ -265,7 +265,7 @@ int icm42605_turn_on_fifo(const struct device *dev)
 		return result;
 	}
 
-	fifo_en |= (BIT_FIFO_ACCEL_EN  | BIT_FIFO_GYRO_EN | BIT_FIFO_WM_TH);
+	fifo_en = (BIT_FIFO_ACCEL_EN  | BIT_FIFO_GYRO_EN | BIT_FIFO_WM_TH);
```

Comment:
Maybe it make sense to initialize both _fifo_en_ and _int0_en_ at declaration?

Reply:
ok!